codegens/js-fetch/lib/index.js

```diff
@@ -42,7 +42,9 @@ function parseFormData (body, trim) {
   _.forEach(body, function (data) {
     if (!data.disabled) {
       if (data.type === 'file') {
-        bodySnippet += `formdata.append("${sanitize(data.key, trim)}", "${sanitize(data.src, trim)}");\n`;
+        bodySnippet += `// if access to fs, use fs.createReadStream('${data.src}') ` +
```

Comment:
too complicated. We can move this out as an option later. For now, let's assume it's a browser environment and there's no `fs`.
Also, in the `formdata.append` line, there's no space after the first comma, but there is one after the second.